src/main/res/values/strings.xml

```diff
@@ -1022,7 +1022,7 @@
 	<!-- 2018-01-16 -->
 	<string name="lang_fi" translatable="false">Suomi</string>
 	<string name="lang_my" translatable="false">မြန်မာဘာသာ</string>
-	<string name="lang_nb_rNO" translatable="false">Bokmål</string>
+	<string name="lang_nb_rNO" translatable="false">Norwegian Bokmål</string>
```

Comment:
Should this be "norsk"? (the string should be the Norwegian word for the language rather than the English word)

Reply:
That is just something Wikipedia is doing.

Reply:
Not sure what you mean about Wikipedia -- basically all I'm saying is that these language strings should be in the target language, rather than English. For example, `lang_de` is "Deutsch" rather than "German".